src/components/Player/Player.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+
+import styles from './Player.module.scss';
+
+import { addScript } from '#src/utils/dom';
+import { deepCopy } from '#src/utils/collection';
+import type { JWPlayer } from '#types/jwplayer';
+import type { PlaylistItem } from '#types/playlist';
+import useEventCallback from '#src/hooks/useEventCallback';
+import useOttAnalytics from '#src/hooks/useOttAnalytics';
+import { logDev } from '#src/utils/common';
+
+type Props = {
+  playerId?: string | null;
```

Comment:
Shouldn't playerId be required?  Otherwise the script url ends up being funky and the page will probably error.

Reply:
I've updated it, but it went all the way down to the Config type and validation. Not 100% sure if this has other effects. I guess the `player` property should always be required...

Reply:
Indeed it looks like on the dashboard we allow creation of configs without a player ID. These configs wouldn't work with the web app though (unless they forked it and hard coded a player.) @marcovandeveen do you have any idea why player ID is optional?

Reply:
Imho, I say we enforce it to be not null, since the web app won't work without one